<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <node pkg="turtlesim" type="turtlesim_node" name="turtlesim" respawn="true"/>
  <node pkg="agitr" type="pubvel_with_max" name="publish_velocity"/>
  <param name="max_vel" value="3"/>
  <node pkg="agitr" type="set_bg_color" name="set_bg_color"/>
</launch>
